src/main/java/co/cask/plugin/format/MultiTableConf.java

```diff
@@ -48,15 +51,23 @@
 
   @Macro
   @Nullable
-  @Description("A pattern that defines which tables should be read from. " +
-    "Any table whose name matches the pattern will read. If not specified, all tables will be read.")
-  private String tableNamePattern;
+  @Description("A schema name to list all the tables from. By default all the schemas will be used in the listing call.")
+  private String schemaName;
 
   @Nullable
   @Description("The name of the field that holds the table name. " +
     "Must not be the name of any table column that will be read. Defaults to 'tablename'.")
   private String tableNameField;
 
+
+  @Nullable
```

Comment:
this can be macro enabled without changing any other part of the code

Reply:
Updated.